components/Header/index.js

```diff
@@ -19,6 +21,9 @@ export default function Header({ open, onToggle }) {
           <StyledList>
             <StyledLink variant={'text-dark'} href={'/players'} onClick={onToggle}>Show Players</StyledLink>
             <StyledLink variant={'text-dark'} href={'/players/favorites'} onClick={onToggle}>Show Favorites</StyledLink>
+            {session && (
+              <StyledLink variant={'text-dark'} href={'/chat'} onClick={onToggle}>Go to chat</StyledLink>
```

Comment:
I don't see what `onToggle` does here: why do you need a click handler on a next/link which redirects to another page?

Reply:
The toggle function closed the sidebar navigation, is a fallback to reset the state. But it was no chase of this US, only here added al seperate link if user has a session and so there authentificated to see the chat